src/views/preview/preview.scss

```diff
@@ -124,7 +124,7 @@ $stage-width: 480px;
     }
 
     .comments-container {
-        width: 60%;
+        width: 65%;
```

Comment:
Hmm, I think we need to refactor if we are using magic numbers like these... Ideally the margins, padding, and flex-basis would constrain the proportions.